tools/palletjack-tools.gemspec

```diff
@@ -30,6 +30,8 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler", "~> 1.7.8"
   spec.add_development_dependency "rake", "~> 0.9.6"
   spec.add_development_dependency "rspec", "~> 2.14.1"
+  spec.add_development_dependency "rspec_structure_matcher", "~> 0.0.6"
```

Comment:
Perhaps. Everything else I've found is worse than `rspec_structure_matcher`, and it being a development dependency does lessen the impact.